doc/en/goodpractices.rst

```diff
@@ -291,7 +291,7 @@ your own setuptools Test command for invoking pytest.
     setup(
         # ...,
         tests_require=["pytest"],
-        cmdclass={"test": PyTest},
+        cmdclass={"pytest": PyTest},
```

Comment:
Isn't this string (`"test"`/`"pytest"`) the name of the command executed in the next section?

```
python setup.py test
```

Reply:
Earlier in the document, `'test'` is created as an alias to the `'pytest'`:
```ini
[aliases]
test=pytest
```

The command doesn't work unless you remove this from `setup.cfg` or replace that alias with the canonical name in `setup.py`.

Reply:
Ahh right, thanks for the clarification! 👍